docs/android/v2/plugins/Virtual-background.mdx

```diff
@@ -0,0 +1,286 @@
+This guide provides an overview of usage of the Virtual Background plugin of 100ms.
+
+<div style={{textAlign: 'center'}}>
+
+![VirtualBackground](/docs/v2/android-vb.gif)
+
+</div>
+
+
+## Supported Devices/Versions/Resolutions
+
+-   Minimum Android api level required to support Virtual Background plugin is 21, same as that required for 100ms SDK
+-   Minimum 100ms SDK version it can work with is `2.2.2`
+-   Virtual background plugin is built for armeabi-v7a, arm64-v8a, x86 and x86_64 architectures
+-   We have not extended support for 720p and 1080p resolution for this feature
+-   We recommend that you use this feature on a high performance device for smooth experience
+
+## Size Increase Per Architecture
+
+- `x86_64` - 15 MB
+-  `x86`   - 14 MB
+-  `arm64-v8a` - 14.5 MB
+-  `armeabi-v7a` - 11 MB
+
+## Prerequisites
+
+Familiarity with Android Studio and the [fundamentals](https://developer.android.com/training/basics/firstapp) of android apps.
+
+## Common Terms
+
+-   `image background` - The image which the plugin should replace the background with in the user's video.
+-   `plugin load time` - The time taken by the plugin to load the  machine learning model for the first time. The machine learning model is what will identify what part of the image the background is and is essential to virtual background working.
+
+## Add dependency
+
+-   Adding the Virtual Background plugin and sdk dependency to your app-level `build.gradle` [![](https://jitpack.io/v/100mslive/android-sdk.svg)](https://jitpack.io/#100mslive/android-sdk).
+
+```json{3,4}:build.gradle
+dependencies {
+ // See the version in the jitpack badge above.
+ implementation 'com.github.100mslive:android-sdk-virtual-background:x.x.x'
+ implementation 'com.github.100mslive:android-sdk:x.x.x'
+    }
+```
+## Interfaces
+
+### Instantiate VirtualBackground
+
+Instantiate the 100ms Virtual Background plugin like this:
+
+<Tabs id="instanstiate-VirtualBackground" items={['Kotlin', 'Java']} />
+
+<Tab id='instanstiate-VirtualBackground-0'>
+
+```kotlin
+val hmsSdk = HMSSDK
+      .Builder(application)
+      .build()
+val virtualBackground = HMSVirtualBackground(hmsSdk, imageBitmap)
+```
+
+</Tab>
+
+<Tab id='instanstiate-VirtualBackground-1'>
+
+```java
+HMSSDK hmsSdk = new HMSSDK
+                       .Builder(application)
+                       .build();
+HMSVirtualBackground virtualBackground = new HMSVirtualBackground(hmsSdk, imageBitmap)
+```
+</Tab>
+
+That's it. You have instantiated Virtual Background plugin successfully. 🥳
+
+Now let's take a look at the signature of the HMSVirtualBackground.
+
+`HMSVirtualBackground` accepts 2 arguments -
+-   `hmsSdk`: an object of type `HMSSDK` class
+-   `imageBitmap`: A background image bitmap you want to replace your background to.
+
+### Check if plugin is supported
+
+`HMSVirtualBackground.isSupported` can be used to check if the API level/Resolution of device is supported or not.
```

Comment:
What resolution do we check here?

Reply:
We are checking the input video resolution, not supporting if exceeds 480p